src/common/MetaHead.ts

```diff
@@ -1,5 +1,6 @@
 export interface MetaHeadProps {
   type: "metahead"
+  language?: "en"
```

Comment:
nit: can remove this, the head tag would not be able to set the html language sadly